parent/pom.xml

```diff
@@ -29,17 +29,17 @@
         <java.minimum.version>1.8</java.minimum.version>
 
         <!-- core -->
-        <commons-lang3.version>3.4</commons-lang3.version>
+        <commons-lang3.version>3.9</commons-lang3.version>
         <javax-inject.version>1</javax-inject.version>
 
         <!-- module dependencies -->
-        <jackson.version>2.9.4</jackson.version>
-        <guice.version>4.1.0</guice.version>
-        <argparse4j.version>0.4.4</argparse4j.version>
+        <jackson.version>2.10.2</jackson.version>
```

Comment:
Because Jackson had a security vulnerability before 2.9.9, my project team used 2.9.10. It is recommended to update the version to 2.9.10 or 2.10.0.pr3.
```suggestion
        <jackson.version>2.9.10</jackson.version>
```